SourceX/DiabloUI/diabloui.cpp

```diff
@@ -207,7 +209,11 @@ void UiFocus(int itemIndex, bool wrap = false)
 void selhero_CatToName(char *in_buf, char *out_buf, int cnt)
 {
 	std::string output = utf8_to_latin1(in_buf);
+#ifdef USE_SDL1
+	output.resize(1);
+#else
 	output.resize(SDL_TEXTINPUTEVENT_TEXT_SIZE - 1);

```

Comment:
could just imlement SDL_TEXTINPUTEVENT_TEXT_SIZE as a define with a value of 2 (lenght of a C string with one char and on zero terminator)

Reply:
Done